app/src/main/java/net/gsantner/markor/util/AppSettings.java

```diff
@@ -813,7 +813,7 @@ public boolean getSetWebViewFulldrawing(boolean... setValue) {
     }
 
     public String getTodotxtAdditionalContextsAndProjects() {
-        return getString(R.string.pref_key__todotxt__additional_projects_contexts, "+music +video @home @shop").replaceAll("\\s+", " ").trim();
+        return getString(R.string.pref_key__todotxt__additional_projects_contexts, "+music +video @home @shop");
```

Comment:
We don't require the replace and trim code. The additional string is parsed by the same logic which parses tasks so multiple and trailing spaces are ignored.

Reply:
edittext preference is multiline (no android: option). Task class is for single line of text?

Reply:
The task parser can handle multiple lines (i.e. text with `\n`). 

![image](https://user-images.githubusercontent.com/1940940/113917625-43019d80-9796-11eb-9f15-72a4e7ac374e.png)

Works fine